devicon.json

```
@@ -7309,6 +7309,28 @@
         "color": "#903a36",
         "aliases": []
     },
+    {
+        "name": "envoyproxy",
+        "altnames": [
+            "envoy"
+        ],
```

imo these should switch places. The website calls it envoy, and the logo says envoy, so I would think the icon name should be envoy too
```suggestion
        "name": "envoy",
        "altnames": [
            "envoyproxy"
        ],
```